tools/python/boutiques/schema/example_good.json

```diff
@@ -0,0 +1,236 @@
+{
+  "name": "ndmg",
+  "tool-version": "v0.0.41-1",
+  "description": "dwi connectome estimation pipeline",
+  "command-line": "ndmg_pipeline [DTI] [BVAL] [BVEC] [MPRAGE] [ATLAS] [MASK] [OUTDIR] [DESIKAN] [JHU] [TALAIRACH] [AAL] [HARVARDOXFORD] [CPAC200] [CLEAN]",
+  "container-image": {
```

Comment:
you could spell this `containerImage` to make sure that this is not silently ignored (see `invalid-tool.json` in `examples/invalid/typos`).